packages/components/popover/Popover.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: 'Popover'
+type: 'component'
+status: 'stable'
+slug: /components/popover/
+github: 'https://github.com/contentful/forma-36/tree/master/packages/components/popover'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-popover--basic'
+typescript: ./src/Popover.tsx
+---
+
+Popover is used to display some content on top of another, and should be paired with a clickable trigger element.
+It is a base for other more specific component, like `Menu`, `Autocomplete` and `Multiselect`. Please, consider using these specific componenets to cover your needs.
```

Comment:
We (with designers) decided to give the users more flexibility. Since we are removing `Dropdown` component which was something  in the middle between raw popover and specific components like `Menu` or `Multiselect`.

> I guess we have other components like this, right?
What do you mean?

Reply:
Nevermind! hahahaha thanks for clarifying 👌